packages/reporting/src/request/steps/redirect-tagger.js

```diff
@@ -18,10 +20,14 @@ import TempSet from '../temp-set';
 export default class RedirectTagger {
   constructor() {
     this.redirectCache = new TempSet();
-    this.cacheTimeout = 10000;
     this.redirectTaggerCache = new TempSet();
   }
 
+  async init() {
+    await this.redirectCache.isReady;
```

Comment:
I do not see an "isReady" attribute in TempSet. Awaiting the undefined field is a noop, right?